application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue

```diff
@@ -251,6 +251,9 @@ export default {
       if (!newMessage || !newMessage.trim()) {
         return;
       }
+      if (newMessage.match(/<img (alt)?[^>]*>/g)) {
+        newMessage = newMessage.replaceAll(/<img (alt)?[^>]*src="/g,`<img alt="${this.$t('exoplatform.chat.team.msg.img.alt')}" src="`);
```

Comment:
```suggestion
      if (newMessage.match(/<img (alt(='')?)? [^>]*>/g)) {
        newMessage = newMessage.replaceAll(/<img (alt(='')?)?[^>]*src="/g,`<img alt="${this.$t('exoplatform.chat.team.msg.img.alt')}" src="`);
```
That way, you cover the case of `<img alt='' src='...'> ` IMHO, WDYT ?